packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -40,6 +40,14 @@ export type UseResizeHandleParams = {
    * Use this for localization.
    */
   getA11ValueText?: (value: number) => string;
+
+  /**
+   * Only measure relative change in size, useful to use with CSS calc() function.
+   * Example: clamp(60px, calc(20% + var(--nav-size)), 40%)
+   * This will take 20% of the parent size by default, but will allow to resize between 60px and 40%.
+   * Also, the size will still be relative in nature (20% + (X)px).
+   */
+  relative?: boolean;
```

Comment:
The repo now has support for playwright tests, we should have tests there. In this case let's not block you with testing the entire thing but just the new relative feature

Reply:
We agreed that follow up PR will include tests as this is quite time sensitive right now